source/common/upstream/cds_api_impl.cc

```diff
@@ -96,10 +100,13 @@ void CdsApiImpl::onConfigUpdate(const std::vector<Config::DecodedResourceRef>& a
   for (const auto& resource_name : removed_resources) {
     if (cm_.removeCluster(resource_name)) {
       any_applied = true;
-      ENVOY_LOG(info, "cds: remove cluster '{}'", resource_name);
+      ENVOY_LOG(debug, "cds: remove cluster '{}'", resource_name);
     }
   }
 
+  ENVOY_LOG(debug, "cds: added/updated {} cluster(s), {} skipped unmodified cluster(s)",
```

Comment:
Maybe make this one info? I don't think that should be a perf issue?

Also I would suggest but up to you:

```suggestion
  ENVOY_LOG(debug, "cds: added/updated {} cluster(s), skipped {} unmodified cluster(s)",
```

Reply:
done